api/database_cluster_backup.go

```diff
@@ -34,7 +34,7 @@ import (
 const (
 	databaseClusterBackupKind = "databaseclusterbackups"
 
-	storageCleanupFinalizer     = "everest.percona.com/dbb-storage-cleanup"
+	storageProtectionFinalizer  = "everest.percona.com/dbb-storage-protection"
```

Comment:
the constant can be available directly from the `everest-operator` source code, the `everest` repo has it as a dependency anyway, wdyt about using it directly? 

It would require first to update the module to
`go get github.com/percona/everest-operator@EVEREST-979`

and then after the merge of the operator PR we would need to update the module to  
`go get github.com/percona/everest-operator@main` before merging this PR

Reply:
Thanks, updated